WebService/data/device-simulation-template.json

```diff
@@ -0,0 +1,32 @@
+﻿[
```

Comment:
1. I'd move the file under Services/data to avoid confusion
2. the file contains multiple templates, so I'd change the filename suffix to "templates"
3. are these templates just examples? should RM template be here?